search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -785,6 +785,17 @@
   (when-let [c-concept-ids (util/seqify (:collection-concept-id params))]
     (mapcat (partial cc/concept-id-validation :collection-concept-id) c-concept-ids)))
 
+(defn- timeline-collection-identifier-validation
+  "Validates that collection identifier parameters are present for timeline searches.
+   Accepts collection-concept-id, entry-id, entry-title, provider, or short-name + version combination."
+  [_concept-type params]
+  (when-not (or (:collection-concept-id params)
```

Comment:
I notice some queries come in with `echo_collection_id` - do we need to directly handle that here or is there some other translation that occurs earlier which turns that into a collection_concept_id?

Reply:
I was under the impression that echo colleciton id would be the same as collection concept id and was just an old legacy parameter that should just be removed, that is why I didn't add it here.